var/spack/repos/builtin/packages/xkeyboard-config/package.py

```diff
@@ -16,13 +17,26 @@ class XkeyboardConfig(AutotoolsPackage):
 
     version('2.18', sha256='d5c511319a3bd89dc40622a33b51ba41a2c2caad33ee2bfe502363fcc4c3817d')
 
+    variant('xorg',
+            default=False,
+            description='add xorg sysmlinks to base in share/X11/xkb/rules/')
+
     depends_on('libx11@1.4.3:')
 
     depends_on('libxslt', type='build')
     depends_on('pkgconfig', type='build')
     depends_on('intltool@0.30:', type='build')
     depends_on('xproto@7.0.20:', type='build')
 
+    @run_after('install')
+    def symlink_xorg(self):
+        if '+xorg' in self.spec:
+            tty.warn('linking xorg->base in ' +
```

Comment:
Personally I don't think we need to tty.warn here. I think `symlink` already adds a debug message.